tslint.json

```diff
@@ -0,0 +1,39 @@
+{
+  "extends": ["tslint:recommended", "tslint-react", "tslint-config-prettier", "tslint-immutable"],
+  "rules": {
+    "interface-name": false,
+    "member-ordering": false,
+    "no-console": false,
+    "no-debugger": false,
```

Comment:
Shouldn't this be `true`?

Reply:
Good find. `no-console` and `no-debugger` should be `true`. At least for a production build and in the CI.

But in dev mode I ~want~ need at least `debugger`. 😅 
So it would be nice to have a seperate rule set (or rule subset) for `DEV` and `PROD`.

Reply:
But you didn't need to merge it, right? Travis would fail but that would be all.
`console` is actually needed we use it in the extensibility package afaik.
But I think it would be okay to use comments to disable the linter for this file or this specific lines (I think this is possible with tslint, at least it is with eslint). This way you would be aware that you shouldn't use it at least.

WDYT?

Reply:
Good idea.

You can disable tslint like eslint.
https://palantir.github.io/tslint/usage/rule-flags/

So, set `no-console` and `no-debugger` to `true`?

Reply:
Yessss! :)

Reply:
Yeah, guess it's OK to override the rule in those few cases where we legitly need `console.x`.